packages/camera/camera/example/lib/main.dart

```diff
@@ -27,32 +27,33 @@ IconData getCameraLensIcon(CameraLensDirection direction) {
       return Icons.camera_front;
     case CameraLensDirection.external:
       return Icons.camera;
+    default:
+      throw ArgumentError('Unknown lens direction');
   }
-  throw ArgumentError('Unknown lens direction');
 }
 
-void logError(String code, String message) =>
+void logError(String code, String? message) =>
     print('Error: $code\nError Message: $message');
 
 class _CameraExampleHomeState extends State<CameraExampleHome>
     with WidgetsBindingObserver, TickerProviderStateMixin {
-  CameraController controller;
-  XFile imageFile;
-  XFile videoFile;
-  VideoPlayerController videoController;
-  VoidCallback videoPlayerListener;
+  CameraController? controller;
+  XFile? imageFile;
+  XFile? videoFile;
+  VideoPlayerController? videoController;
+  VoidCallback? videoPlayerListener;
   bool enableAudio = true;
   double _minAvailableExposureOffset = 0.0;
   double _maxAvailableExposureOffset = 0.0;
   double _currentExposureOffset = 0.0;
-  AnimationController _flashModeControlRowAnimationController;
-  Animation<double> _flashModeControlRowAnimation;
-  AnimationController _exposureModeControlRowAnimationController;
-  Animation<double> _exposureModeControlRowAnimation;
-  AnimationController _focusModeControlRowAnimationController;
-  Animation<double> _focusModeControlRowAnimation;
-  double _minAvailableZoom;
-  double _maxAvailableZoom;
+  AnimationController? _flashModeControlRowAnimationController;
+  Animation<double>? _flashModeControlRowAnimation;
+  AnimationController? _exposureModeControlRowAnimationController;
+  Animation<double>? _exposureModeControlRowAnimation;
+  AnimationController? _focusModeControlRowAnimationController;
+  Animation<double>? _focusModeControlRowAnimation;
```

Comment:
It looks like all of these could be `late` since they are initialized in `initState`.